e2e-tests/src/driver.rs

```diff
@@ -41,6 +41,14 @@ pub struct TestContext<'a> {
     pub proposal_id: Option<String>,
 }
 
+#[derive(Debug, Hash, Clone, clap::ValueEnum, Copy, PartialEq, Eq)]
+pub enum Protocol {
+    Ethereum,
+    Near,
+    Stellar,
+    Icp,
+}
```

Comment:
~nit: I find it best to keep things that are exposed to the CLI all in one place, as it's easy to forget the `clap::ValueEnum` flag

so this should still be in `main.rs`, and if you have reasons to not have it there, it should at the very least be in `protocol.rs`